swagger.yaml

```diff
@@ -1654,7 +1654,8 @@ paths:
           description: Overrides the ISO type for the disovery ignition, either 'full-iso' or 'minimal-iso'.
           required: false
           type: string
-          enum: ['full-iso', 'minimal-iso']
+          enum: ['use-infraenv-type', 'full-iso', 'minimal-iso']
```

Comment:
`use-infraenv-type` shouldn't be necessary here. If it isn't specified in the url we should use the type from the infra-env.
That also ensures backwards compatibility.

Reply:
Actually I think it would be best if you could do something like this here:

```yaml
- in: query
  name: discovery_iso_type
  description: ...
  required: false
  schema:
    $ref: '#/definitions/image_type'
```